src/libpgexporter/extension.c

```diff
@@ -380,4 +380,66 @@ pgexporter_load_single_extension_yaml(char* extensions_path, char* extension_nam
       fclose(file);
    }
    return 1;
+}
+
+bool
+pgexporter_extension_in_list(char* extension_name, char* extensions_list)
```

Comment:
There's a style of writing that a lot of C codebases use, where if a string they take in as a parameter remains unchanged, they mark it as `const char *` (while `char * const` being for pointers that remain unchanged, and their combinations).

The compiler has no problem in either, and a good one can figure out the implicit const, but it's for the developer's benefit.